.travis.yml

```diff
@@ -0,0 +1,14 @@
+sudo: required
+
+language: generic
+
+services:
+  - docker
+
+before_install:
+  - docker build  .
```

Comment:
Is there any more value here than checking if the plugin compiles? If that's all there is, I don't think it's worth the 30 minute Travis build...

Reply:
I added threading warnings as errors, so we can catch some warnings in prs as well now )